DatadogSessionReplay/SRSnapshotTests/SRHost/Fixtures/UnsupportedViewsViewController.swift

```diff
@@ -0,0 +1,39 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import UIKit
+
+internal class UnsupportedViewsViewController: UIViewController {
+    
+    @IBOutlet weak var swiftUIContainer: UIView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let hostingController = UIHostingController(rootView: TestView())
+        addChild(hostingController)
+        swiftUIContainer.addSubview(hostingController.view)
+        hostingController.view.frame = swiftUIContainer.bounds
+        hostingController.didMove(toParent: self)
+    }
+}
+
+import SwiftUI
+
+struct TestView: View {
```

Comment:
nit/ It's a generic name and will be confusing in internal namespace. Because we use it only in this file:
```suggestion
fileprivate struct TestView: View {
```